bin/update

```diff
@@ -28,18 +40,20 @@ chdir APP_ROOT do
   # If we need a "reset all the things" script, we should call it bin/reset.
   system! 'bundle update'
 
+  # load 'Rakefile'
+  load_rakefile
   puts "\n== Updating database =="
-  system! 'bin/rails db:migrate'
+  Rake::Task["db:migrate"].invoke
 
   puts "\n== Seeding database =="
-  system! "bin/rails db:seed GOOD_MIGRATIONS=skip"
+  Rake::Task["db:seed"].invoke
```

Comment:
Again, see the "Considerations" section of the PR description.

I agree, and this allow would be a reason not to merge this PR.  I am aware of it, but wanted to show the benefits at the very least as a POC.